parsl/app/bash.py

```diff
@@ -108,13 +108,17 @@ def __init__(self, func, data_flow_kernel=None, cache=False, executors='all', ig
             if sig.parameters[s].default is not Parameter.empty:
                 self.kwargs[s] = sig.parameters[s].default
 
-        # update_wrapper allows remote_side_bash_executor to masquerade as self.func
-        # partial is used to attach the first arg the "func" to the remote_side_bash_executor
-        # this is done to avoid passing a function type in the args which parsl.serializer
-        # doesn't support
-        remote_fn = partial(update_wrapper(remote_side_bash_executor, self.func), self.func)
-        remote_fn.__name__ = self.func.__name__
-        self.wrapped_remote_function = wrap_error(remote_fn)
+        if 'BalsamExecutor' not in executors:
```

Comment:
I'm concerned that this check would break support for multiple executors within a single run.

Reply:
Elaborate? Maybe give me a test case to implement?